default-templates/new-account/index.html

```diff
@@ -122,17 +151,100 @@ <h1>Apps</h1>
   function render() {
     const { session } = state
     const actionButton = sessionActionButton()
+    const logged_href = sessionLogged_href()
     if (!session) {
       actionButton.innerText = 'Log in'
+      logged_href.style.display = 'none'
+      logged_href.href = "";
+      logged_href.title = "";
     } else {
       actionButton.innerText = 'Log out'
+      if (session.webId) {
+        logged_href.style.display = 'initial'
+        logged_href.href = session.webId;
+        logged_href.title = session.webId;
+      }
     }
```

Comment:
A bit too much magic here. Why not separate Log In and Log Out buttons that we show and hide, instead of one multipurpose button?

Reply:
The "magic" was added for show user icon(like anchor with href=webId)  on the left side of Login/Logout button.  The code for Login/Logout button was already in the original version.